knowledge-base/displaying-reports-from-report-server-through-custom-report-source-resolver.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Displaying Reports From Report Server Through Custom Report Source Resolver
+description: Explains how to connect to Report Server through Custom Report Source Resolver and display a reprot
+type: how-to
+page_title: How to Display Reports From Report Server through Custom Report Source Resolver
+slug: displaying-reports-from-report-server-through-custom-report-source-resolver
+position: 
+tags: 
+ticketid: 1519100
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+Sometimes you want to avoid adding the Report Server's credentials in the initialization of the viewer. This can be done by the usage of [Custom Report Source Resolver](../telerik-reporting-rest-custom-report-resolver).
+
+## Solution
```

Comment:
Add also that we use the HTTP Client Telerik.ReportServer.HttpClient that is distributed with the Report Server